docs/user/apirule-migration/01-83-migrate-jwt-v1beta1-to-v2.md

```diff
@@ -0,0 +1,154 @@
+# Migrate APIRule `v1beta1` of Type **jwt** to Version `v2`
+
+Learn how to migrate an APIRule created in version `v1beta1` using the **jwt** handler to version `v2`.
+
+## Context
+
+APIRule in version `v1beta1` is deprecated and scheduled for removal. Once the APIRule custom resource definition (CRD) stops serving version `v1beta1`, the API server will no longer respond to requests for APIRules in this version. As a result, you will encounter errors when attempting to access the APIRule custom resource using the deprecated `v1beta1` version. Therefore, migrating to version `v2` is required.
+
+## Prerequisites
+
+* You have the Istio and API Gateway modules added.
+* You have a deployed workload exposed by an APIRule in the deprecated `v1beta1` version. The APIRule uses the **jwt** handler.
+  > [!NOTE] 
+  > The workload exposed by the APIRule in version `v2` must be a part of the Istio service mesh. See [Enable Istio Sidecar Proxy Injection](https://kyma-project.io/#/istio/user/tutorials/01-40-enable-sidecar-injection?id=enable-istio-sidecar-proxy-injection).
+* You have installed [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) and [yq](https://mikefarah.gitbook.io/yq).
+
+## Steps
+
+This example demonstrates a migration from an APIRule `v1beta1` with the **jwt** handler to an APIRule `v2` with the **jwt** handler.
+The example uses an HTTPBin service, exposing the `/anything` and `/.*` endpoints. The HTTPBin service is deployed in its own namespace, with Istio enabled, ensuring the workload is part of the Istio service mesh.
+
+1. Obtain a configuration of the APIRule in version `v1beta1` and save it for further modifications. For instructions, see [Retrieve the Complete **spec** of an APIRule in Version `v1beta1`](./01-81-retrieve-v1beta1-spec.md). See a sample of the retrieved **spec** in the YAML format:
```

Comment:
```suggestion
1. Retrieve a configuration of the APIRule in version `v1beta1` and save it for further modifications. For instructions, see [Retrieve the Complete **spec** of an APIRule in Version `v1beta1`](./01-81-retrieve-v1beta1-spec.md). See a sample of the retrieved **spec** in the YAML format:
```

I believe obtain, retrieve (and get) all mean the same in this context, right? I'd use the same term everywhere for consistency; and I'd prefer the simplest vocabulary possible (obtain < retrieve < get); maybe "retrieve" is a good middle ground because you use it in the other document title anyway?